website/versioned_docs/version-v3.9.x/gator.md

```diff
@@ -13,33 +12,40 @@ To install `gator`, you may either
 relevant to your system or build it directly from source.
 
 To build from source:
+
 ```
 go get github.com/open-policy-agent/gatekeeper/cmd/gator
 ```
 
 ## The `gator test` subcommand
 
 `gator test` allows users to test a set of Kubernetes objects against a set of
-Templates and Constraints.  The command returns violations when found and
+Templates and Constraints. The command returns violations when found and
 communicates success or failure via its exit status.
 
-Note: The `gator verify` command was first called `gator test`.  These names
-were changed to better align `gator` with other projects in the
-open-policy-agent space.
+The `gator test` command will also attempt to expand any resources passed in, if
+a supplied `ExpansionTemplate` matches these resources. If a resource supplied
```

Comment:
> If a resource supplied to `gator test` has a custom namespace, and the config for that namespace is supplied, `gator test` can match any relevant namespace data.

IMO This may be superfluous, since users would just expect this to work

Reply:
Removed.